src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+import duke.tasks.Task;
+
+
+/**
+ * UI class that is responsible for reading user input and printing
+ */
+public class Ui {
+    /**
+     * Constructor to initialize the UI Class
+     */
+    public Ui() {
+    }

```

Comment:
If the class is not storing any variables, it might not be needed to initiate an instance of it, instead, you can just make all the methods static